package.json

```diff
@@ -40,6 +40,7 @@
     "versions:uikit": "manypkg upgrade @commercetools-uikit"
   },
   "dependencies": {
+    "@babel/helper-define-map": "^7.12.13",
```

Comment:
No idea why, but apparently this dep somehow does not get installed with Yarn.

Reply:
Can you explain what it has to do with the context of this PR?  I can't even find documentation.

Reply:
Nothing to do with this PR. On my machine this dep simply didn't get installed, which prevents the website to start.